src/components/charts/chart.ts

```diff
@@ -560,10 +563,8 @@ export default class AstraChart extends ClassifiedElement {
       let firstRecordValue = firstRecord ? firstRecord[this.keyX ?? ''] : ''
 
       if (firstRecordValue && !isNaN(Number(firstRecordValue))) {
-        let number = Number(firstRecordValue)
-
-        // Show number with commas
-        firstRecordValue = number.toLocaleString()
+        const number = parseFloat(`${firstRecordValue}`)
```

Comment:
something seems off that we're casting it to a string, then parsing a float out of it? what was it before casting to a string?

Reply:
It's an `any`, it could be of any type being returned from the database row